model/outgoing_webhook_test.go

```diff
@@ -5,142 +5,92 @@ package model
 
 import (
 	"net/url"
-	"reflect"
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestOutgoingWebhookJson(t *testing.T) {
 	o := OutgoingWebhook{Id: NewId()}
 	json := o.ToJson()
 	ro := OutgoingWebhookFromJson(strings.NewReader(json))
 
-	if o.Id != ro.Id {
-		t.Fatal("Ids do not match")
-	}
+	assert.Equal(t, o.Id, ro.Id, "Ids do not match")
 }
 
 func TestOutgoingWebhookIsValid(t *testing.T) {
 	o := OutgoingWebhook{}
-
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	assert.NotNil(t, o.IsValid(), `empty declaration should be invalid`)
 
 	o.Id = NewId()
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	assert.NotNil(t, o.IsValid(), `Id = NewId should be invalid`)
```

Comment:
Here and on more cases below it might be good to use `assert.NotNilf` and print out the actual value that caused the failure instead of an [hardcoded](https://en.wikipedia.org/wiki/Hard_coding) one.